src/components/Card.js

```diff
@@ -43,11 +83,13 @@ export default class Card {
     this._cardDeleteButton = this._cardElement.querySelector(
       ".card__delete-button"
     );
+    this._cardLikeCounter = this._cardElement.querySelector(".card__counter");
```

Comment:
Good job that you've found these variables only once here and defined them as **class fields**